frontend/public/components/_dropdown.scss

```diff
@@ -166,6 +166,22 @@ $color-bookmarker: #DDD;
   }
 }
 
+.co-import-yaml-link {
```

Comment:
This CSS addition should probably live somewhere other than _dropdown.scss since these elements are not dropdown related?

Reply:
the link lives in the co-namespace-selector dropdown. Would it be better to create a new scss file for the link?

Reply:
That'd be my vote.  IMO, the styles in _dropdown.scss be specific to dropdowns.